examples/helloworld/src/index.js

```diff
@@ -1,4 +1,26 @@
 import React from 'react';
+import _ from 'underscore';
 import App from './App';
 
-React.render(<App />, document.getElementById('root'))
+
+var state = {
+    very: {
+        deeply: {
+            nested: {
+                counts: _.range(400).map(function () { return 0; })
+            }
+        }
+    }
+};
+
+function swapper(f) {
```

Comment:
It could be better to wrap the `swapper` and `queueRender` functions to a Cursor function, e.g., `Cursor.render`. That function accepts the initial state and the React render callback.

```
var cmp = Cursor.render(
    initialState,
    cursor => React.render(<Component cursor={cursor} />, dom));
```